controllers/depresolver/depresolver_config.go

```diff
@@ -257,13 +350,13 @@ func parseLogOutputFormat(value string) LogFormat {
 func (c *Config) GetExternalClusterNSNames() (m map[string]string) {
 	m = make(map[string]string, len(c.ExtClustersGeoTags))
 	for _, tag := range c.ExtClustersGeoTags {
-		m[tag] = getNsName(tag, c.DNSZone, c.EdgeDNSZone, c.EdgeDNSServer)
+		m[tag] = getNsName(tag, c.DNSZone, c.EdgeDNSZone, c.EdgeDNSServers[0].Host)
```

Comment:
What will happen if first ([0]) edge dns server is not accessible?

Reply:
inside the `getNsName()` the dns server is used only in case it's equal to `localhost` or `127.0.0.1` (then it's returned -> dummy for tests) otherwise the dns server is ignored and the return value is constructed using `EDGE_DNS_ZONE`, geotag and what not.

After the validation it's guaranteed that there will be at least 1 element in that array so `c.EdgeDNSServers[0].Host` can't fail.

Reply:
@ytsarev , @jkremser  - here is the discussion about the topic: https://github.com/k8gb-io/k8gb/pull/605#discussion_r709180510